src/Features/CSharp/Portable/CodeFixes/SimplifyTypeNames/SimplifyTypeNamesCodeFixProvider.cs

```diff
@@ -87,9 +89,11 @@ internal static string GetCodeActionId(string diagnosticId, string nodeText)
             switch (diagnosticId)
```

Comment:
"GetCodeActionId" is called with node.ToString().  can you call "ConvertToSingleLine" on the node first so that we won't get any weird displays with trivia.